cfgov/v1/tests/wagtail/test_organisms.py

```diff
@@ -0,0 +1,224 @@
+		self.assertContains(response, '123 abc street')
+
+	def test_sidebar_contact_info(self):
+		"""Full width text content correctly displays on a Learn Page"""
+		learn_page = LearnPage(
+				title='Learn Page', 
+				slug='learn', 
+		)
+		learn_page.content = StreamValue(
+			learn_page.content.stream_block, 
+			[atomic.full_width_text],
+			True
+		)
+		publish_page(child=learn_page)
+		response = c.get('/learn/')
+		self.assertContains(response, 'Full width text content')
+
+
+	def test_image_text_groups(self):
+		"""Image Text Groups correctly display on a Landing Page"""
+		landing_page = LandingPage(
+			title='Landing Page', 
+			slug='landing',
+		)
+		landing_page.content = StreamValue(
+			landing_page.content.stream_block,
+			[
+				atomic.image_text_50_50_group,
+				atomic.image_text_25_75_group
+			],
+			True,
+		)
+		publish_page(child=landing_page)
+		response = c.get('/landing/')
+		self.assertContains(response, 'Image 25 75 Group')
+		self.assertContains(response, 'Image 50 50 Group')
+
+
+	def test_half_width_link_blob_group(self):
+		"""Half width link blob group correctly displays on a Landing Page"""
+		landing_page = LandingPage(
+			title='Landing Page', 
+			slug='landing', 
+		)
+		landing_page.content = StreamValue(
+			landing_page.content.stream_block,
+			[atomic.half_width_link_blob_group],
+			True
+		)
+		publish_page(child=landing_page)
+		response = c.get('/landing/')
+		self.assertContains(response, 'Half Width Link Blob Group')
+
+	# TODO: More comprehensive test for this organism
+	def test_email_signup(self):
+		"""Email signup correctly displays on a Sublanding Page"""
+		sublanding_page = SublandingPage(
+			title='Sublanding Page', 
+			slug='sublanding', 
+		)
+		sublanding_page.sidefoot = StreamValue(
+			sublanding_page.sidefoot.stream_block,
+			[atomic.email_signup],
+			True
+		)
+		publish_page(child=sublanding_page)
+		response = c.get('/sublanding/')
+		self.assertContains(response, 'Email Sign up')
+
+	# TODO: More comprehensive test for this organism
+	def test_reg_comment(self):
+		"""RegComment correctly displays on a Sublanding Page"""
+		sublanding_page = SublandingPage(
+			title='Sublanding Page', 
+			slug='sublanding', 
+		)
+		sublanding_page.content = StreamValue(
+			sublanding_page.content.stream_block,
+			[atomic.reg_comment],
+			True
+		)
+		publish_page(child=sublanding_page)
+		response = c.get('/sublanding/')
+		self.assertContains(response, 'Enter your comments')
+
+	def test_table(self):
+		"""Table correctly displays on a Learn Page"""
+		learn_page = LearnPage(
+				title='Learn Page', 
+				slug='learn', 
+		)
+		learn_page.content = StreamValue(
+			learn_page.content.stream_block, 
+			[atomic.table],
+			True
+		)
+		publish_page(child=learn_page)
+		response = c.get('/learn/')
+		self.assertContains(response, 'table hyperlink')
+		self.assertContains(response, 'table text')
+		self.assertContains(response, 'table text blob')
```

Comment:
Ah yes, thanks.  I took these assertions from the previous code and didn't notice that.